all-clouds/all-clouds-explanation/ubuntu-base-and-minimal-images.rst

```diff
@@ -0,0 +1,48 @@
+Ubuntu Base and Minimal Images
+==============================
+
+Broadly speaking, CPC produces two types of images - a base image and a minimal
+image.
+
```

Comment:
Please provide links to where people can download these images or where they can discover them

Reply:
Please also share that both Base and Minimal are available for all clouds too for all Ubuntu releases.